src/SFA.DAS.Provider.PR.Application/Services/EmailCheckingService.cs

```diff
@@ -19,15 +20,10 @@ public static bool IsValidDomain(string? email)
             return false;
         }
 
-        try
-        {
-            var hostEntry = Dns.GetHostEntry(domain);
+        var lookup = new LookupClient();
 
-            return hostEntry.AddressList.Length > 0;
```

Comment:
I think we should use the async look up here

Reply:
ok